client/auth/native-login/index.js

```diff
@@ -0,0 +1,29 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import Gridicon from 'gridicons';
+
+/**
+ * Internal dependencies
+ */
+import Main from 'components/main';
+import LoginBlock from 'blocks/login';
+import { localize } from 'i18n-calypso';
+
+const Login = ( { translate } ) => (
+	<Main className="native-login">
+		<div>
+			<div className="native-login__header">
+				<Gridicon icon="user-circle" size={ 72 } />
+				<div>{ translate( 'You are signed out' ) }</div>
+			</div>
+			<div className="native-login__container">
+				<LoginBlock
+					title={ translate( 'Sign in to WordPress.com' ) } />
+			</div>
+		</div>
```

Comment:
Checking if there is some CSS involved here, I think I used some code from `signup`.